Dockerfile

```diff
@@ -8,13 +8,14 @@ RUN apt-get update -y && \
     libgeos-dev \
     libproj-dev \
     libspatialindex-dev \
+    libudunits2-dev \
     proj-bin \
     proj-data
 
 ENV CPLUS_INCLUDE_PATH /usr/include/gdal
 
 # Install R packages
-RUN install2.r --error --ncpus 4 --deps TRUE tidyverse shiny leaflet rgdal raster shinydashboard DT shinyjs
+RUN install2.r --error --ncpus 4 --deps TRUE dplyr shiny leaflet raster shinydashboard DT shinyjs sf
```

Comment:
you can use the `dependecies.R` file instead here

Reply:
1e953783542b6a52710f05bbc0cb9dc4dca6ab7b